Podfile

```diff
@@ -1,5 +1,8 @@
 platform :ios, '9.0'
 
+# ignore all warnings from all pods
+inhibit_all_warnings!
```

Comment:
I feel so much lol

Reply:
Is this alright though? Not really much we can do about the warnings pods are raising, and they are really annoying (I find) when trying to look at my own errors.